docs/source/quickinstall.rst

```diff
@@ -8,39 +8,52 @@ This quick install guide outlines the basic steps needed to install OpenMC on
 your computer. For more detailed instructions on configuring and installing
 OpenMC, see :ref:`usersguide_install` in the User's Manual.
 
-----------------------------------------
-Installing on Linux/Mac with conda-forge
-----------------------------------------
-
-`Conda <https://conda.io/en/latest/>`_ is an open source package management
-system and environment management system for installing multiple versions of
-software packages and their dependencies and switching easily between them. If
-you have `conda` installed on your system, OpenMC can be installed via the
+--------------------------------------------------
+Installing on Linux/Mac with Mamba and conda-forge
+--------------------------------------------------
+
+`Conda <https://conda.io/en/latest/>`_ and `Mamba <https://mamba.readthedocs.io/en/latest/>`_
+are an open source package management systems and environments management
+system for installing multiple versions of software packages and their
+dependencies and switching easily between them. If you have `conda` installed
```

Comment:
I would split up these two sentences: one describing conda and another describing mamba.

Reply:
Is it worth adding links to conda installers such as Miniconda, AnaConda and Miniforge (which is open source :+1: )